From 1f36dc7b024f450c5448f6aac665cc712d1576d5 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sun, 27 Oct 2019 14:42:41 +0200
Subject: [PATCH] [SW-0] goya2: add basic Makefile

Creates empty arcboot.bin to be copied to BINARIES_DIR
This complete the needed framework
Now need to populate with SNPS SDK and customize it to our requierments

Change-Id: I0024e9940ac7db8e3187911753af87ac03d1a376
---

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..2c985f1
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,4 @@
+output
+.cproject
+.project
+.settings/
diff --git a/Makefile b/Makefile
new file mode 100644
index 0000000..c897beb
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,2 @@
+all:
+	@touch arcboot.bin
\ No newline at end of file
